crates/transparency/src/map/proof.rs

```diff
@@ -34,23 +34,23 @@ where
     V: VisitBytes,
 {
     value: PhantomData<V>,
-    peers: Vec<Option<Hash<D>>>,
+    peers: Vec<Hash<D>>,
```

Comment:
Ironically, this might want to become `Option<..>` again but for a different reason. If we ever mark something as `None` that can mean that the client knows it's an empty subtree and can substitute `empty_tree_hash(height)` for the value.